.github/workflows/continuous_integration.yml

```
@@ -0,0 +1,47 @@
+name: "Continuous Integration"
```

```suggestion
---
name: "Continuous Integration"
```

Nit (`yamllint` loves document separators)